Jmeter/UserStory3.jmx

```diff
@@ -0,0 +1,1407 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.6.3">
+  <hashTree>
+    <TestPlan guiclass="TestPlanGui" testclass="TestPlan" testname="Test Plan">
+      <elementProp name="TestPlan.user_defined_variables" elementType="Arguments" guiclass="ArgumentsPanel" testclass="Arguments" testname="User Defined Variables">
```

Comment:
your performance script is not correctly written you should only one thread group with test fragments containing all the requests! you have also not come up with the flow properly! please see the video that i have link to, to show how to correctly do this! this should be torwards the end of that talk!

Reply:
also use a stepping thread group https://jmeter-plugins.org/wiki/PluginsManager/
this is so you can download it